content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -475,28 +475,29 @@ validation and auto-completion.
 
 ## Top-level plot definitions
 
-The list of `plots` contains one or more user-defined top-level plots (paths
-relative to the location of `dvc.yaml`).
+The list of `plots` contains one or more user-defined `dvc plots`
+configurations. Every plot needs a unique file path (relative to the location of
+`dvc.yaml`) as identifier. Optional
+[configuration fields](#available-configuration-fields) can be provided as well.
 
-Every plot has to have its own ID. Configuration, if provided, should be a
-dictionary.
-
-In the simplest use case, a user can provide the file path as the plot ID and
-not provide configuration at all:
+In the simplest use, you only need to provide the plot's file path. In the
+example below, DVC will take data from `logs.csv` and use the default plotting
+behavior (apply the `linear` plot [template] to the last found column):
 
 ```yaml
 # dvc.yaml
 ---
 plots:
-  logs.csv:
+logs.csv:
 ```
 
-In that case the default behavior will be applied. DVC will take data from
-`logs.csv` file and apply `linear` plot
-[template](/doc/user-guide/visualizing-plots#plot-templates-data-series-only) to
-the last found column (CSV, TSV files) or field (JSON, YAML).
+[template]: /doc/user-guide/visualizing-plots#plot-templates-data-series-only
 
-We can customize the plot by adding appropriate fields to the configuration:
+For customization, we can use a plot ID (`confusion_matrix` below). This is
+displayed in as title, unless we override it with a `title` value. We also
+specify columns within the data source file (`confusion_matrix_data.csv`) for
+the `y` and `x` axes. Finally, the built-in template for confusion matrices is
+set.
 
 ```yaml
 # dvc.yaml
```

Comment:
The main issue with this section in the dvc.yaml spec is that it's in form of a guide. It should be a bit more dry like a table or something, as this page is for the plots file spec/ formalities. Maybe the **Available configuration fields** section is enough here and the rest can be mixed into the existing guide text and/or simplified and moved (back 😅) to the cmd ref.